### DISCUSSION TITLE
Task 03 - Question 1 (Typo)

### DISCUSSION THREAD

**Discussion by hastishwan-uw** [chulminy/CIVE497-CIVE700#155] [Q&A]
![MicrosoftTeams-image](https://github.com/chulminy/CIVE497-CIVE700/assets/48458021/c2ec90e1-6a22-495b-bb48-34a4027b6cf1)

Just wondering is it a typo ( Q1 is f(t) and g(t) )  or both of them is F(t) ?

**Comment by chulminy** [comment]
I think it's typo. The bottom one is g(t).
